dll/win32/kernel32/client/sysinfo.c

```diff
@@ -393,17 +393,8 @@ GetNumaProcessorNode(IN UCHAR Processor,
  * @param AvailableBytes
  *        The amount of available memory for the node, in bytes.
  *
- * @return If the function succeeds, the return value is non zero.
- *         If the function fails, the return value is zero. To get
- *         extended error information, call GetLastError.
+ * @return TRUE or FALSE on error.You can get extended information using the GetLastError function.
  *
- * @remarks The GetNumaAvailableMemoryNode function returns the amount of memory consumed
```

Comment:
I'm not sure what the parts about the memory performance counters mean, so I didn't bother with them, but we should mention that single node systems and multi-node systems are calculated differently.
```
  * @remarks GetNumaAvailableMemoryNode returns how much memory is in free and
             zeroed pages on the node. If a systems has only one node, the
             return value will include standby pages. If a system has more than
             one node, the return value will not include standby pages.
```